ext_localconf.php

```diff
@@ -239,7 +239,6 @@
     'PageView',
     [
         \Kitodo\Dlf\Controller\PageViewController::class => 'main',
```

Comment:
Müsste die Action dann nicht hier hinzugefügt werden? Denn unter den "non-cacheable actions" taucht sie ja weiterhin auf.

```suggestion
        \Kitodo\Dlf\Controller\PageViewController::class => 'main, addDocument',
```